content/en/bits_ai/managing_incidents.md

```diff
@@ -59,33 +56,22 @@ Without going into the Datadog web app, you can ask Bits AI in Slack to:
 
 ## Generate a first draft of a postmortem
 
-<div class="alert alert-info">AI-assisted postmortem drafting is in private beta.</div>
-
 To generate an AI-assisted postmortem draft:
 
-1. Navigate to the incident page in Datadog. For example, to view Incident 2679, you might search for `2679` on the [**Incidents**][2] page and click the relevant match.
-1. Ensure that the incident is resolved and that the timeline has ten or more messages.
-1. Click the **Generate Postmortem** button.
-1. Select the AI template.
-1. Allow up to one minute for the postmortem to be generated. Do not close the tab during this time.
-1. Review the generated postmortem. AI-generated postmortems serve as a first draft to help your incident responders get started, and they require human revisions.
-
-## Replace an existing postmortem
+1. Navigate to the incident which you'd like to generate a postmortem for in Datadog. 
+2. Ensure that the incident is resolved and that its timeline has ten or more messages.
+3. Click the **Generate Postmortem** button.
+4. Select the either the out-of-the-box **General incident with AI content** template or a [custom template](#customize-postmortem-templates-with-ai-incident-variables) that you've created. 
+6. Allow up to one minute for the postmortem to be generated. Do not close the tab during this time.
+7. Review the generated postmortem. AI-generated postmortems serve as a first draft to help your incident responders get started, and they require human revisions.
 
-If an incident already has a linked postmortem, you can unlink it before [generating a new postmortem](#generate-a-first-draft-of-a-postmortem). All postmortems are still accessible in the incident's Notebook.
+## Customize postmortem templates with AI incident variables 
 
-1. Navigate to the incident page in Datadog. For example, to view Incident 2679, you might search for `2679` on the [**Incidents**][2] page and click the relevant match.
-1. Click **Postmortem**.
-1. Click the trash icon.
-
-## Customize the postmortem template
-
-1. Navigate to [**Service Mgmt > Incident > Settings > Postmortems**][1].
-1. Click **New Postmortem Template** and customize your template using the provided incident variables.  
-    - Variables prefixed by `ai`, such as `incident.ai_action_items`, yield AI-generated content instead of fixed values.
+1. Navigate to [**Service Mgmt > Incidents > Settings > Postmortems**][1].
+2. Click **New Postmortem Template** and customize your template using the provided incident variables.  
+    - Variables prefixed by `ai` yield AI-generated content instead of fixed values. There are six `ai` variables: `{{incident.ai_action_items}}`, `{{incident.ai_customer_impact}}`, `{{incident.ai_key_timeline}}`, `{{incident.ai_lessons_learned}}`, `{{incident.ai_summary}}`, and `{{incident.ai_system_overview}}`. 
```

Comment:
```suggestion
   - Variables prefixed by `ai` yield AI-generated content instead of fixed values. There are six `ai` variables: `{{incident.ai_action_items}}`, `{{incident.ai_customer_impact}}`, `{{incident.ai_key_timeline}}`, `{{incident.ai_lessons_learned}}`, `{{incident.ai_summary}}`, and `{{incident.ai_system_overview}}`. 
```